.github/workflows/jit.yml

```diff
@@ -87,18 +85,10 @@ jobs:
             architecture: x86_64
             runner: ubuntu-22.04
             compiler: gcc
-          - target: x86_64-unknown-linux-gnu/clang
-            architecture: x86_64
-            runner: ubuntu-22.04
-            compiler: clang
           - target: aarch64-unknown-linux-gnu/gcc
             architecture: aarch64
             runner: ubuntu-22.04
             compiler: gcc
-          - target: aarch64-unknown-linux-gnu/clang
-            architecture: aarch64
-            runner: ubuntu-22.04
-            compiler: clang
     env:
       CC: ${{ matrix.compiler }}
```

Comment:
I don't think we need this anymore (in fact, do we need `compiler` in the matrix above at all?):

```suggestion
```